### PR TITLE
Changed main node paramaters and some logging details.

### PR DESCRIPTION
**Pull request by Fraser999**

### CODE REVIEW

File: src/event.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ use self::Event::*;
 pub enum Event {
     AddNode(Name),
     RemoveNode(Name),
-    RemoveNodeFrom(Prefix),
+    RemoveNodeFrom(Prefix),
```
Comment (fizyk20) [inline src/event.rs]: Suddenly - trailing whitespace (I'm guessing that's just rustfmt acting up, though).

File: src/node.rs [hunk 2]
```diff
@@ -114,7 +133,7 @@ impl Node {
                      compute_current_candidate_blocks(potentially_current));
         self.current_blocks = compute_current_blocks(&self.current_candidate_blocks);
         if is_ambiguous(&self.current_blocks, self.our_name) {
-            self.ambiguous_step = self.ambiguous_step.or(Some(step));
+            self.ambiguous_step = self.ambiguous_step.or_else(|| Some(step));
```
Comment (fizyk20) [inline src/node.rs]: That's a Clippy suggestion I didn't think made sense - does it?

File: src/event.rs (hunk 1 above)
Comment (Fraser999) [inline src/event.rs]: Yeah - I'll fix it!

File: src/node.rs (hunk 2 above)
Comment (Fraser999) [inline src/node.rs]: Yes, it's this [clippy lint](https://github.com/Manishearth/rust-clippy/wiki#or_fun_call).  I think it does make sense, but is hardly worth bothering about given the cost of the allocation here and that this is a simulator.

**Pull request merged**